bigframes/_config/bigquery_options.py

```diff
@@ -91,6 +91,7 @@ def __init__(
         skip_bq_connection_check: bool = False,
         *,
         ordering_mode: Literal["strict", "partial"] = "strict",
+        client_endpoints_override: dict = {},
```

Comment:
quick question, why not just endpoints_override.

Reply:
Those are bq_xxx_client endpoints.